sensor_msgs/CompressedImage observed_img
int32 count
string mode
---
bool success